README.md

```diff
@@ -2,26 +2,24 @@
 
 Welcome to the Nava Platform! This is the home repository for Nava's Platform tools that empowers teams to accelerate their delivery through self-service tools and training.
 
```

Comment:
I think it should be "that empower teams" because it's referring to "platform tools" not "repository"

Reply:
fixed